### DISCUSSION TITLE
[Run issue]: Installation on WSL

### DISCUSSION THREAD

**Discussion by Geoffvinc** [topas-nbio/TOPAS-nBio#9] [Run-Problem] [answered]
### Contact Details

geoffvinc2004@gmail.com

### What was the issue?

I installed TOPAS-nBio on WSL following the installation guide for DEBIAN/UBUNTU. However, I cannot run the extension on TOPAS and it returns the output "Sorry, don't have the logic yet to add a TsCharltonDNA" when I choose the geometries from the extension. Does anyone has any idea on solving this problem?

### TOPAS-nBio category

Geometry

### Version

3.0

### What Operative System (OS) are you running on?

Windows (WSL)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment]
Did you get any compilation errors while installing TOPAS-nBio (after installing OpenTOPAS) or miss the `make install` command?

**Reply by Geoffvinc** [reply]
Hi @JoseRamosMendez , there are no error when compiling TOPAS-nBio  and I didn't miss the make install command. However, there are warnings when running the cmake command as shown below
```
CMake Warning at cmake/Modules/TOPAS_Options.cmake:9 (message):
  Setting CMAKE_BUILD_TYPE=Release
Call Stack (most recent call first):
  CMakeLists.txt:21 (include)
```
I wonder if this affect the installation.

**Reply by Geoffvinc** [reply, marked as answer]
Hi, I figured out that the issue only persist when adding the geometry in useQT GUI. On the other hand, adding geometry from the script would not have any issue. Thank you for the help!
